components/EntryForm.js

```diff
@@ -0,0 +1,72 @@
+import styled from "styled-components";
+import DatePicker from "react-datepicker";
+import { useState } from "react";
+import "react-datepicker/dist/react-datepicker.css";
```

Comment:
I recommend to do your own styling here using styled-components and not to mix different approaches.